source/features/next-prev-notifications.tsx

```diff
@@ -0,0 +1,26 @@
+import './next-prev-notifications.css';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager';
+import observe from '../helpers/selector-observer';
+
+function addNextPreviousBtn(notificationsHeader: Element): void {
+	notificationsHeader.parentElement?.parentElement?.classList.add('flex-items-between');
```

Comment:
Use `.closest(some selector)` and never `.parentElement`

- https://github.com/sindresorhus/eslint-plugin-unicorn/issues/345